sd_dynamic_prompts/dynamic_prompting.py

```diff
@@ -28,12 +29,16 @@
 )
 from sd_dynamic_prompts.pnginfo_saver import PngInfoSaver
 from sd_dynamic_prompts.prompt_writer import PromptWriter
-from sd_dynamic_prompts.utils import get_logger
 
 VERSION = __version__
 
-logger = get_logger(__name__)
-logger.info(f"Starting Dynamic Prompts v{VERSION}")
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
```

Comment:
Given your thoughts on the module setting its own level - shouldn't this line be removed?